documentation/blog/2025-04-21-practical-use-cases-of-ai/index.md

```diff
@@ -0,0 +1,281 @@
+
+### How to Try It
+
+1. Enable the [Developer](/extensions/detail?id=developer) extension
+2. Prompt Goose:
+   > What does this phrase mean: “Who does Vegas have as the favorite?”
+
+---
+
+## 6. 📊 Querying a Relational Database
+
+### Use Case
+
+I asked Goose for insights about my data using natural language, and it wrote a Common Table Expression for me.
+
+### Why It’s Useful
+
+SQL can get complex with joins, stored procedures, and subqueries. Goose helps me move faster and avoid errors by handling the query logic for me. 
+
+### How to Try It
+
+1. Follow [this tutorial](/docs/tutorials/postgres-mcp)
+2. Enable the [PostgreSQL](https://github.com/modelcontextprotocol/servers/tree/HEAD/src/postgres) and [Developer](/extensions/detail?id=developer) extensions
+3. Prompt Goose:
+   > Search my database for any students whose GPA is higher than the average in their grade level. Return name, grade, GPA difference, and the average GPA.
+
+<iframe class="aspect-ratio" src="https://www.youtube.com/embed/PZlYQ5IthYM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+---
+
+## 7. 🗓 Planning My Conference Speaking Strategy
+
+### Use Case
+
+I've used Goose to analyze historical conference data so I could plan smarter for upcoming CFP deadlines.
+
+### Why It’s Useful
+
+I tend to overbook myself or get anxious that I won't get accepted, so I apply to everything. Then I end up getting accepted to all of them and say yes without thinking, which leads to poor planning and rushed talks. With Goose, I can analyze patterns in CFP timelines and make more intentional choices.
+
+### How to Try It
+
+1. Follow [this tutorial](/docs/tutorials/agentql-mcp)
+2. Enable the [AgentQL](https://github.com/tinyfish-io/agentql-mcp) extension
+3. Prompt Goose:
+   > I'm a tech conference speaker planning my 2025-2026 submissions. 
+   > Extract for developer conferences (attendance > 500) occurring between 2022-2024:
+   > - Conference name
+   > - Conference dates
+   > - CFP timeline 
+   >
+   > To identify:
+   > - Consistent monthly patterns
+   > - Whether conferences stick to same months yearly
+   > - If CFP windows are consistent year-to-year
+   > - Any shifts in traditional timing
+   >
+   > Structure results as JSON
+
+---
+
+## 8. 🐞 Tracking Down a Buggy Commit
+
+### Use Case
+
+A feature broke, but I had made so many commits, I couldn't tell which one introduced the bug. I asked Goose to help me run git bisect, so we could identify the problematic code.
+
+### Why It’s Useful
+
+The hardest part of debugging is often just figuring out where to look. Git bisect makes that faster, and Goose walked me through the process without needing to memorize the steps.
+### How to Try It
```

Comment:
double-check that your spacing between things is consistent. Here, you don't have a blank line between 182 and 183, but up at line 30 you have two blank lines between. Pretty picky detail, just my own brain tracking that kinda stuff